tasks/sass.js

```diff
@@ -1,8 +1,15 @@
 var gulp = require('gulp');
 var sass = require('gulp-sass');
 var config = require('../gulp.config')();
+var replace = require('gulp-replace');
  
-gulp.task('sass', function () {
+gulp.task('set-theme', function () {
+    return gulp.src(config.assetsPath.styles + 'bootswatch.scss')
+        .pipe(replace(/bootswatch\/[^\/]+\//g, 'bootswatch/'+ config.theme + '/'))
+        .pipe(gulp.dest(config.assetsPath.styles));
+});
```

Comment:
Yeah, setting theme every time is not needed, normally. I agree. I'll try using `yargs`, thank you.